yarn.lock

```diff
@@ -16525,6 +16540,12 @@ has-ansi@^2.0.0:
   dependencies:
     ansi-regex "^2.0.0"
 
+has-binary2@cypress-io/has-binary#8580a33df21e8b36a43f57872a82c60829636a92:
```

Comment:
gleb, plz delete your yarn cache so you don't have to suffer, it's hurting me to watch this

Reply:
#%&*##%$#$%%$ this yarn cache lock thing with has-binary2

Reply:
for me, `rm -rf ~/.cache/yarn/` was all i needed to do. maybe it's different on mac

Reply:
by deleting the yarn cache you mean the global yarn cache on my machine?

Reply:
yeah, that's where it's located for me

Reply:
and then "normal" yarn install in the monorepo would not bring the craziness back?

Reply:
Let's try deleting the cache
```
$ yarn cache dir
/Users/gleb/Library/Caches/Yarn/v6
~/git/cypress-docker-images on master
$ rm -rf /Users/gleb/Library/Caches/Yarn/v6
```

Reply:
ok, removed the cache - do I need to rebuild `yarn.lock` file here for this PR?

Reply:
yeah, this has the bad version of has-binary